tests/upgrades/test_performance_tuning.py

```diff
@@ -6,7 +6,9 @@
 
 :CaseLevel: Acceptance
 
-:CaseComponent: Performance
+:CaseComponent: ForemanMaintain
```

Comment:
ForemanMaintain is not definitely the right component for tuning tests, and as Installer is used for setting up a tuning, so it seems the right component, and we should check with @jameerpathan111 to confirm.

Reply:
I agree here. This file is testing Installer.

Reply:
@Gauravtalreja1 I referred other tuning tests to come to this conclusion. Mainly https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/maintain/test_upgrade.py#L9 

I could change my conclusion to installer if you all agree!

Reply:
@Gauravtalreja1 @Griffin-Sullivan Updated to `Installer`, please re-review!